codalab/lib/cli_util.py

```diff
@@ -46,8 +48,13 @@ def desugar_command(orig_target_spec, command):
     val2key = {}  # e.g., a.txt => b1 (use first key)
 
     def get(dep):  # Return the key
-        if ':' in dep:  # :<bundle_spec> or <key>:<bundle_spec>
-            key, val = dep.split(':', 1)
+        if '::' in dep:
```

Comment:
Can we consolidate the spec parsing logic and factor this out with the `parse_target` and `parse_target_specs` methods?